WebApiCoreSeed.WebApi.Tests/Controllers/UserControllerTest.cs

```diff
@@ -0,0 +1,260 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+using WebApiCoreSeed.WebApi.Controllers;
+using WebApiCoreSeed.WebApi.Models;
+using Xunit;
+
+namespace WebApiCoreSeed.WebApi.Tests.Controllers
+{
+    public class UserControllerTest
+    {
+        #region Fields
+
+        private Mock<IUserService> _userService;
+
+        private UserController _classUnderTest;
+
+        #endregion
+
+        #region Constructors
+
+        public UserControllerTest()
+        {
+            _userService = new Mock<IUserService>();
+            _classUnderTest = new UserController(_userService.Object);
+        }
+
+        #endregion
+
+        #region GetAll Tests
+
+        [Fact]
+        public async void GetAll_ShouldReturnAListOfUsers_WhenHasAtLeastOneUser()
```

Comment:
let's use number 5 condition i think is quite clear https://vitalflux.com/7-popular-unit-test-naming-conventions/

Reply:
for all test, let's use the same naming condition

Reply:
Actually I have to disagree @mravinale. @Dongata is using here the convention established by netcore team that also seems to become quite standard along different projects. See https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/testing?view=aspnetcore-2.1. It's very similar but it places the method under test as the first part.

Reply:
Oh man, my comment reads awful 🤣 -- please read with the best of intentions!

Reply:
I'm using the standard that was on the others UT's. Even though, Making Sense has "{TargetTestMethod}_{Expectation}[_When{Condition}]", that, IMHO, it's a good approximation c: